features/safe-area-inset.yml

```
@@ -0,0 +1,8 @@
+name: Safe area inset environment variables
+description: The `safe-area-inset-*` CSS environment variables represent an area of a non-rectangular screen that will not be cut off. For example, `top:\ env(safe-area-inset-top);` positions the top of an element below the notch on an iPhone.
```

There's some discussion on how exactly to formulate it into a guideline in https://github.com/web-platform-dx/web-features/issues/1388, but in general, we want avoid putting things into code text that a developer wouldn't actually type, so I'm suggesting not using the wild card asterisk here and let the example do the work.

Also, reading the spec and seeing the mention of a rectangle within a non-rectangle really helped me understand what was going on here. What do you think of this amendment?

```suggestion
description: The `safe-area-inset-` CSS environment variables represent a rectangle that cannot cut off content within a non-rectangular screen. For example, `top:\ env(safe-area-inset-top);` positions the top of an element below the notch on an iPhone.
```